multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakSettings.java

```diff
@@ -109,10 +118,16 @@ public KeycloakSettings(
     LOG.info("openid configuration = {}", openIdConfiguration);
 
     Map<String, String> settings = Maps.newHashMap();
+    Map<String, String> internalSettings = Maps.newHashMap();
     settings.put(
         USERNAME_CLAIM_SETTING, usernameClaim == null ? DEFAULT_USERNAME_CLAIM : usernameClaim);
     settings.put(CLIENT_ID_SETTING, clientId);
     settings.put(REALM_SETTING, realm);
+
+    if (serverInternalURL != null) {
```

Comment:
is that possible that `serverInternalURL` is null?

Reply:
Yes, it is. Using internal network will be like an option.

Reply:
I think only che-operator installer will set up this env variable.

Reply:
what for this information is exposed over HTTP?

Reply:
serverURL will be exposed in the https://github.com/eclipse/che/blob/master/multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakConfigurationService.java#L50 

serverInternalURL - won't be exposed.